TARGET_STM32WL/TARGET_STM32WLE5xC/TARGET_LORA_E5_BREAKOUT/PinNames.h

```diff
@@ -0,0 +1,153 @@
+/* mbed Microcontroller Library
+ * SPDX-License-Identifier: BSD-3-Clause
+ ******************************************************************************
+ *
+ * Copyright (c) 2016-2020 STMicroelectronics.
+ * All rights reserved.
+ *
+ * This software component is licensed by ST under BSD 3-Clause license,
+ * the "License"; You may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *                        opensource.org/licenses/BSD-3-Clause
+ *
+ ******************************************************************************
+ *
+ * Automatically generated from STM32CubeMX/db/mcu/STM32WL55JCIx.xml
+ */
+
+#ifndef MBED_PINNAMES_H
+#define MBED_PINNAMES_H
+
+#include "cmsis.h"
+#include "PinNamesTypes.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef enum {
+    ALT0  = 0x100,
+    ALT1  = 0x200,
+    ALT2  = 0x300,
+    ALT3  = 0x400,
+    ALT4  = 0x500
+} ALTx;
+
+typedef enum {
+
+    PA_0       = 0x00,
+    PA_1       = 0x01,
+    PA_1_ALT0  = PA_1  | ALT0, // same pin used for alternate HW
+    PA_2       = 0x02,
+    PA_2_ALT0  = PA_2  | ALT0, // same pin used for alternate HW
+    PA_3       = 0x03,
+    PA_3_ALT0  = PA_3  | ALT0, // same pin used for alternate HW
+    PA_4       = 0x04,
+    PA_5       = 0x05,
+    PA_6       = 0x06,
+    PA_7       = 0x07,
+    PA_7_ALT0  = PA_7  | ALT0, // same pin used for alternate HW
+    PA_8       = 0x08,
+    PA_9       = 0x09,
+    PA_10      = 0x0A,
+    PA_11      = 0x0B,
+    PA_12      = 0x0C,
+    PA_13      = 0x0D,
+    PA_14      = 0x0E,
+    PA_15      = 0x0F,
+    PB_0       = 0x10,
+    PB_1       = 0x11,
+    PB_2       = 0x12,
+    PB_3       = 0x13,
+    PB_4       = 0x14,
+    PB_5       = 0x15,
+    PB_6       = 0x16,
+    PB_7       = 0x17,
+    PB_8       = 0x18,
+    PB_8_ALT0  = PB_8  | ALT0, // same pin used for alternate HW
+    PB_9       = 0x19,
+    PB_9_ALT0  = PB_9  | ALT0, // same pin used for alternate HW
+    PB_10      = 0x1A,
+    PB_11      = 0x1B,
+    PB_12      = 0x1C,
+    PB_13      = 0x1D,
+    PB_14      = 0x1E,
+    PB_15      = 0x1F,
+    PC_0       = 0x20,
+    PC_1       = 0x21,
+    PC_2       = 0x22,
+    PC_3       = 0x23,
+    PC_4       = 0x24,
+    PC_5       = 0x25,
+    PC_6       = 0x26,
+    PC_13      = 0x2D,
+    PC_14      = 0x2E,
+    PC_15      = 0x2F,
+    PH_3       = 0x73,
+
+    /**** ADC internal channels ****/
+
+    ADC_TEMP = 0xF0, // Internal pin virtual value
+    ADC_VREF = 0xF1, // Internal pin virtual value
+    ADC_VBAT = 0xF2, // Internal pin virtual value
+
+    // STDIO for console print
+#ifdef MBED_CONF_TARGET_STDIO_UART_TX
+    CONSOLE_TX = MBED_CONF_TARGET_STDIO_UART_TX,
+#else
+    CONSOLE_TX = PB_6,
+#endif
+#ifdef MBED_CONF_TARGET_STDIO_UART_RX
+    CONSOLE_RX = MBED_CONF_TARGET_STDIO_UART_RX,
+#else
+    CONSOLE_RX = PB_7,
+#endif
+
+// Legacy I2C aliases
+#ifndef I2C_SDA
+#define I2C_SDA PA_15
```

Comment:
Is defining alias for I2C and SPI useful?
(same question for all pinnames.h files)

Reply:
Console are not same serial in LoRa-E5 and RAK3172 (thanks guys) so no way, should be in `PinNames.h`
To have same code compiling and working on different targets, I would like to keep alias if you don't mind.
Since it's comited to this branch not sure how to squash, never done that.